src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -16,77 +16,59 @@ extern "C" {
 #include "monodroid.h"
 #include "dylib-mono.h"
 #include "util.h"
-#include "unzip.h"
-#include "ioapi.h"
 #include "embedded-assemblies.h"
 #include "globals.h"
 #include "monodroid-glue.h"
 
+namespace xamarin { namespace android { namespace internal {
+	struct TypeMappingInfo {
```

Comment:
We're in a `.cc` file.  Why's this need to be within a namespace instead of global scope?

(If this were a header file, sure, but implementation?)

Reply:
The `struct `is declared in the header file as being in this namespace so it has to be defined in the same namespace